### DISCUSSION TITLE
how to pass the type meta from java to python, so python needn't to declare @dataclass for deserialization?

### DISCUSSION THREAD

**Discussion by justincui** [apache/fory#1714] [Q&A]
in some case, python has no way to know data class before hand. It's better for java to transmit the data class type meta to python before python start deserialization. Are there any example code for this case?

**Comment by chaokunyang** [comment]
We will do that in future, it has be defined in xlang serialization spec: https://fury.apache.org/docs/specification/fury_xlang_serialization_spec#schema-evolution

**Reply by justincui** [reply]
请问有中文文档么，网上找了半天就找到一点点英文文档，中文的都是那篇说Fury开源了的新闻，没找到开发文档。

**Comment by chaokunyang** [comment]
When creating fory with compatible mode, you can deserialize java objects in python with inconsist schema:

```java
Fory fory = Fory.builder().withXlang(true).withCompatible(true)
```

```python
fory = pyfory.Fory(xlang=True, compatible=True)
```
